engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -33,6 +33,7 @@ paladin_t::paladin_t( sim_t* sim, util::string_view name, race_e r )
 {
   active_consecration = nullptr;
   active_hallow       = nullptr;
+  active_h_hallow     = nullptr;
```

Comment:
I'd find a more explicit name because I had to dig through the code to find out what it stores. Maybe active_hallow_healing?